File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/11.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
	float peso1, peso2;
	string nome1, nome2;
	
	printf("Digite o nome da primeira pessoa: ");
	scanf("%s", &nome1);
	
	printf("Digite o seu peso de %s (Ex. 80): ", &nome1);
	scanf("%f", &peso1);
	
	printf("Digite o nome da segunda pessoa: ");
	scanf("%s", &nome2);
	
	printf("Digite o seu peso de %s (Ex. 80): ", &nome2);
	scanf("%f", &peso2);
	
	if(peso1 > peso2){
		printf("%s e mais pessado do que %s.", &nome1, &nome2);
	} else if(peso1 == peso2){
		printf("%s e %s tem o mesmo peso.", &nome1, &nome2);
	} else {
		printf("%s e mais pessado do que %s.", &nome2, &nome1);
	}
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/Python/11.py
"""
11. Elabore um algoritmo que leia o nome e o peso de duas pessoas e imprima o nome da
pessoa mais pesada
"""

pessoa1 = str(input("Digite o nome da primeira pessoa: "))
peso1 = int(input("Digite o peso de " + pessoa1 + ": "))

pessoa2 = str(input("Digite o nome da segunda pessoa: "))
peso2 = int(input("Digite o peso de " + pessoa2 + ": "))

if peso1 > peso2:
    print(pessoa1 + " é mais pesado(a) que " + pessoa2)
else:
    print(pessoa2 + " é mais pesado(a) que " + pessoa1)